positron/modules/atom_common_v8_util.js

```diff
@@ -11,3 +11,8 @@ exports.getHiddenValue = util.getHiddenValue;
 exports.setHiddenValue = util.setHiddenValue;
 
 exports.setDestructor = positronUtil.makeStub('atom_common_v8_util.setDestructor');
+
+exports.createDoubleIDWeakMap = function() {
+  // TODO: Properly implment this.
+  return new WeakMap();
+};
```

Comment:
Do we actually need this yet? I think this is only needed once we upgrade Electron.